avatars/processors/expected_mean.py

```diff
@@ -24,11 +23,41 @@ def __init__(
         ---------
             target_variables:
                 variables to transform
+
+        Keyword Arguments
+        -----------------
             groupby_variables:
                 variables to use to group values in different distributions
             same_std:
                 Set to True to force the variables to transform to have the same
                 standard deviation as the reference data. default: False.
+
+        Examples
+        --------
+        >>> import numpy as np
+        >>> df = pd.DataFrame(np.array(([1, 2, 3], [4, 5, 6], [4, 5, 6], [1, 2, 3])),
+        ...                   index=['mouse', 'rabbit', 'horse', 'cat'],
+        ...                   columns=['one', 'two', 'three'])
+        >>> df = df.astype('int')
+        >>> processor = ExpectedMeanProcessor(target_variables = ['one'])
+        >>> processed = processor.preprocess(df)
+
+        The processor save the mean and the std of each target variables
+
+        >>> processor.properties_df
+          ___NOGROUPVAR___  onemean    onestd
+        0   __NOGROUPVAL__      2.5  1.732051
+        
+        Now you can force your synthetic dataset to have the same mean as the original.
+        
+        >>> avatar = pd.DataFrame(np.array(([3, 2, 3], [3, 5, 6], [8, 5, 6], [8, 2, 3])),
+        ...                   index=['mouse', 'rabbit', 'horse', 'cat'],
+        ...                   columns=['one', 'two', 'three'])
+        >>> avatar.one.mean()
+        5.5
+        >>> avatar = processor.postprocess(df, avatar)
+        >>> avatar.one.mean()
+        2.5
```

Comment:
This is very nice for a doc to have this type of detailed example, especially because the example is tested. But isn't there a risk to have a very long docstring for complex processors or functions? Expected mean is a good example where we could add a groupby variable, resulting in another example like the one you wrote here. Just wondering

Reply:
it's a food question and I don't think it is an issue. 
To compare, you can have a look to the pandas doc , it is sometimes long (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.replace.html)

Reply:
I'm also wondering what the test files of the processors will be like if a good example is given and tested in the docstring !